app/components/Form/CustomTitles.tsx

```diff
@@ -53,27 +55,57 @@ const IndigenousEntity = () => {
   );
 };
 
-const NumberOfHouseholds = () => {
+const GeographicArea = () => {
+  return (
+    <div>
+      Referring to the{' '}
+      <Link
+        href="https://catalogue.data.gov.bc.ca/dataset/8fbfc57f-4381-4a10-a4e8-0f335c6fe39a/resource/e12440ae-0b5d-40dc-af4f-2098f0b9374f"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        project zones
+      </Link>{' '}
+      (application guide Annex 6), which zone(s) will this project be conducted
+      in?
+    </div>
+  );
+};
+
+const SupportingDocuments = () => {
   return (
     <div>
-      Final number of Eligible Households targeted by this proposal. This value
-      should match cell G50 in Template 1 – Eligibility Summary.
+      Please upload additional attachments. Please avoid using special
+      characters in the file name. Files must be less than 100mb. If you must
```

Comment:
```suggestion
      characters in the file name. Files must be less than 100MB. If you must
```
100 millibits is a bit small